vue/src/modules/Workspace/view/timeline/TimelineForm.vue

```diff
@@ -39,8 +38,12 @@ export default {
     }
   },
   computed: {
-    ...mapState(['loading', 'showModal', 'modalTimeline']),
-    ...mapStateWorkspace(['workspaces']),
+    ...mapWorkspaceState({
+      loading: state => state.view.timeline.loading,
```

Comment:
@skavt Why form modal is using timeline list loading component ? It should have it's respective loading attribute, `state.view.timeline.modal.loading`